site/.env.template

```diff
@@ -24,6 +24,7 @@ BIGCOMMERCE_STORE_API_CLIENT_SECRET=
 
 NEXT_PUBLIC_SHOPIFY_STORE_DOMAIN=
 NEXT_PUBLIC_SHOPIFY_STOREFRONT_ACCESS_TOKEN=
+NEXT_PUBLIC_SHOPIFY_ADMIN_ACCESS_TOKEN=
```

Comment:
same here, we don't want admin token to be public

Reply:
same typo again, to be corrected.